### DISCUSSION TITLE
Set-Theme : The term 'Set-Theme' is not recognized as the name of a cmdlet, function, script file, or operable program.

### DISCUSSION THREAD

**Discussion by JanDeDobbeleer** [JanDeDobbeleer/oh-my-posh#460] [Troubleshoot] [answered]
Set-Theme : The term 'Set-Theme' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\HP\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1:3 char:1
+ Set-Theme Paradox
+ ~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Set-Theme:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

**Comment by JanDeDobbeleer** [comment, marked as answer]
You need to migrate V2 to V3 using the following [guide](https://ohmyposh.dev/docs/upgrading).

The quick fix is to replace `Set-Theme` with `Set-PoshPrompt`, but it's advised to read the guide.

**Reply by blQSheep** [reply]
Thank you! i went trough the given guide and it works.
The windows documentation also has been updated now.

**Reply by patridge** [reply]
For anyone else who runs into this issue and misinterprets what this answer is suggesting, `Set-PoshPrompt` seems to be the new theme command going forward. You may not have had V2 installed, but [whatever reference you were using](https://www.hanselman.com/blog/how-to-make-a-pretty-prompt-in-windows-terminal-with-powerline-nerd-fonts-cascadia-code-wsl-and-ohmyposh) had `Set-Theme`, which is the older command. Switching to `Set-PoshPrompt` will likely work going forward, but the guide will include any other changes that may affect the guide you are using.

When I found this discussion while trying to fix the same issue, I thought it was suggesting I somehow had oh-my-posh V2 installed without knowing it. At that point, I thought you were suggesting that `Set-PoshPrompt` was some backwards-compatibility legacy workaround command and `Set-Theme` would start working again after following the migration guide. That seems to be completely wrong and `Set-PoshPrompt` is the correct command going forward for all folks. The suggestion to read the guide is simply so you know about all the possibly related changes to V3 that you might encounter when following along with older guides.

**Reply by mahiuddin-dev** [reply]
> You need to migrate V2 to V3 using the following [guide](https://ohmyposh.dev/docs/upgrading).
> 
> The quick fix is to replace `Set-Theme` with `Set-PoshPrompt`, but it's advised to read the guide.
Thanks a lot
